docs/KNOWN_ISSUES.md

```diff
@@ -93,3 +93,12 @@ When casting a player to AirPlay while other players are playing other content l
 ### Workaround
 
 Pause players playing content locally.
+
+## Presentation size can't be determined when AirPlay is enable (FB12080967)
+
+When AirPlay is activated, the value of the `AVPlayerItem.presentationSize` property is consistently zero.
+Therefore, we encounter the challenge of discerning whether the player content corresponds to a video or audio format when AirPlay is on.
```

Comment:
```suggestion
It is therefore not possible to tell whether a video or audio is played when AirPlay is enabled.
```

Reply:
Since both sentences are also related the newline between them should probably be dropped.